cmd/talosctl/cmd/talos/output/output.go

```diff
@@ -23,19 +25,25 @@ type Writer interface {
 
 // NewWriter builds writer from type.
 func NewWriter(format string) (Writer, error) {
-	switch format {
-	case "table":
-		return NewTable(), nil
-	case "yaml":
-		return NewYAML(), nil
-	case "json":
-		return NewJSON(), nil
+	writer := os.Stdout
+
+	switch {
+	case format == "table":
+		return NewTable(writer), nil
+	case format == "yaml":
+		return NewYAML(writer), nil
+	case format == "json":
+		return NewJSON(writer), nil
+	case strings.HasPrefix(format, "jsonpath"):
```

Comment:
```suggestion
	case strings.HasPrefix(format, "jsonpath="):
```

prefix should be same as the index expression below, otherwise it might be off by one error